angular/src/lib/toggle-switch/toggle-switch.component.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, OnInit, Input, Output, EventEmitter, ChangeDetectorRef, forwardRef } from '@angular/core';
+import { NG_VALUE_ACCESSOR, ControlValueAccessor } from '@angular/forms';
+
+const CUSTOM_TOGGLE_SWITCH_VALUE_ACCESSOR: any = {
+  provide: NG_VALUE_ACCESSOR,
+  useExisting: forwardRef(() => ToggleSwitchComponent),
+  multi: true
+};
+
+@Component({
+  selector: 'cui-toggle-switch',
+  template: `
+    <div
+      class="cui-input-group cui-toggle-switch"
+      [ngClass]="wrapperClasses"
+    >
```

Comment:
Why not remove this div wrapper and bind `class="cui-input-group cui-toggle-switch"` to host element. So that wrapperClasses is not needed.

Reply:
removed wrapper and switched to @HostBinding.